packages/alert/src/AlertBase.tsx

```diff
@@ -15,7 +15,7 @@ export const AlertWrapper = styled.div<Partial<AlertProps>>`
   border: 0;
   box-sizing: border-box;
   padding: 8px;
-  max-width: 800px;
+  max-width: ${(props) => (props.type === 'relative' ? '100%' : '800px')};
```

Comment:
Can you change this to use `rm`? Or do we use px in this repo?

Reply:
We use px in this repo